test/helpers/manifest_generator.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"path"
+	"text/template"
+)
+
+var manifestBase = "k8sT/manifests"
+
+// ManifestValues wraps manifest index
+type ManifestValues struct {
+	Index int
+}
+
+// GenerateManifestForEndpoints generates k8s manifests that will create
+// endpointCount cilium endpoints when applied.
+// 1/3 of endpoints is going to be servers, the rest clients.
+// returns lastServer index
+func GenerateManifestForEndpoints(endpointCount int, manifestPath string) (string, int, error) {
+	configMapStr, err := ioutil.ReadFile(path.Join(manifestBase, "html.yaml"))
+	if err != nil {
+		return "", 0, err
+	}
+
+	serverTemplateStr, err := ioutil.ReadFile(path.Join(manifestBase, "server.yaml"))
```

Comment:
If you can move  `server.yaml`, `client.yaml` to `helpers/cons.go` it will be great.